pkg/eventhandler/eventhandlers.go

```diff
@@ -173,6 +175,9 @@ func (eh *EventHandler) startK8sJob(action *config.Action, jsonEventData interfa
 		}
 	}
 
+	// The action ID is generated in such a way that it is unique to the current context of the event
+	actionID := fmt.Sprintf("%d", actionIndex)
```

Comment:
We have the action name, why are we using the position in a list as ID ?
If it's to distinguish actions with the same name we should probably enforce the uniqueness of names or compose some other ID based on it instead of using an index which may change from one version of job config to another...

Reply:
Currently we actually don't try to distinguish actions at all, we just take the first one that matches the event, which also means that the name is currently not required to be unique. If we can make the action name unique that would be a better way to identify the action.

Reply:
@christian-kreuzberger-dtx  What do you think about enforcing unique names that can be used as label values for the actions? How much of a breaking change would it be ?